OIPA/iati/activity_manager.py

```diff
@@ -347,16 +342,16 @@ def prefetch_legacy_data(self):
     def prefetch_conditions(self):
         from iati.models import Conditions, Narrative
 
-        condition_prefetch = Prefetch(
-            'condition__narrative',
-            queryset=Narrative.objects.all()
-            .select_related('language'))
+        # condition_prefetch = Prefetch(
+        #     'condition__narratives',
+        #     queryset=Narrative.objects.all()
+        #     .select_related('language'))
 
         return self.prefetch_related(
             Prefetch(
                 'conditions_set',
                 queryset=Conditions.objects.all()
-                .prefetch_related(condition_prefetch)
```

Comment:
Conditions prefetch is generating only 2 queries without prefetch!
I suggest to make XML export test with multiple Condition and see the impact on Conditions prefetch